### ISSUE TITLE
"    return _C.create_encoding(self.n_input_dims, self.encoding_config, self.precision) RuntimeError: Invalid encoding type: SequentialGrid"

### ISSUE THREAD

**Issue by rockywind** [hturki/suds#6]
Hi, 
Thank you for sharing a great work!
I met the error below!
The train script is `python suds/train.py suds --experiment-name kitti_06 --pipeline.datamanager.dataparser.metadata_path metadata/kitti-06.json` 

````
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
[07:39:19] Saving config to: outputs/kitti_06/suds/2023-04-02_073919/config.yml                 experiment_config.py:129
[07:39:19] Saving checkpoints to: outputs/kitti_06/suds/2023-04-02_073919/nerfstudio_models               trainer.py:128
Time difference quantiles: tensor([0.0074, 0.0074, 0.0074, 0.0074, 0.0074, 0.0074, 0.0074, 0.0074, 0.0074,
        0.0074, 0.0074])
Traceback (most recent call last):
  File "suds/train.py", line 111, in <module>
    suds_entrypoint()
  File "suds/train.py", line 102, in suds_entrypoint
    main(
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/scripts/train.py", line 233, in main
    launch(
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/scripts/train.py", line 172, in launch
    main_func(local_rank=0, world_size=world_size, config=config)
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/scripts/train.py", line 86, in train_loop
    trainer.setup()
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/nerfstudio/engine/trainer.py", line 141, in setup
    self.pipeline = self.config.pipeline.setup(
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/nerfstudio/configs/base_config.py", line 57, in setup
    return self._target(self, **kwargs)
  File "/rockywin.wang/NeRF/suds/suds/data/suds_pipeline.py", line 65, in __init__
    super().__init__(config, device, test_mode, world_size, local_rank)
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/nerfstudio/pipelines/base_pipeline.py", line 236, in __init__
    self._model = config.model.setup(
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/nerfstudio/configs/base_config.py", line 57, in setup
    return self._target(self, **kwargs)
  File "/rockywin.wang/NeRF/suds/suds/suds_model.py", line 350, in __init__
    self.dynamic = DynamicField(
  File "/rockywin.wang/NeRF/suds/suds/fields/dynamic_field.py", line 48, in __init__
    self.encoding = tcnn.Encoding(
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/tinycudann/modules.py", line 292, in __init__
    super(Encoding, self).__init__(seed=seed)
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/tinycudann/modules.py", line 134, in __init__
    self.native_tcnn_module = self._native_tcnn_module()
  File "/opt/conda/envs/suds_my/lib/python3.8/site-packages/tinycudann/modules.py", line 297, in _native_tcnn_module
    return _C.create_encoding(self.n_input_dims, self.encoding_config, self.precision)
RuntimeError: Invalid encoding type: SequentialGrid
````

**Comment by hturki** [comment]
hi - can you make sure you're buidling from the appropriate branch of my tiny-cuda-nn fork? see https://github.com/hturki/suds/issues/1

**Comment by rockywind** [comment]
Hi,
This is my buingding log.
The building script is `pip install git+https://github.com/hturki/tiny-cuda-nn.git@ht/res-grid#subdirectory=bindings/torch`
````
(suds_my) root@bl5a0rj6b5t0e-0:/rockywin.wang/NeRF/suds# pip install git+https://github.com/hturki/tiny-cuda-nn.git@ht/res-grid#subdirectory=bindings/torch -i https://pypi.tuna.tsinghua.edu.cn/simple
Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
Collecting git+https://github.com/hturki/tiny-cuda-nn.git@ht/res-grid#subdirectory=bindings/torch
  Cloning https://github.com/hturki/tiny-cuda-nn.git (to revision ht/res-grid) to /tmp/pip-req-build-_cjnqsex
  Running command git clone --filter=blob:none --quiet https://github.com/hturki/tiny-cuda-nn.git /tmp/pip-req-build-_cjnqsex
  Running command git checkout -b ht/res-grid --track origin/ht/res-grid
  Switched to a new branch 'ht/res-grid'
  Branch 'ht/res-grid' set up to track remote branch 'ht/res-grid' from 'origin'.
  Resolved https://github.com/hturki/tiny-cuda-nn.git to commit 1255fdc87ca3bc0e0fcae0c817438347ee6b751b
  Running command git submodule update --init --recursive -q
  Preparing metadata (setup.py) ... done
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

[notice] A new release of pip available: 23.0 -> 23.0.1
[notice] To update, run: pip install --upgrade pip
````
